target/linux/ipq806x/base-files/etc/board.d/02_network

```diff
@@ -24,6 +25,17 @@ db149)
 	ucidef_add_switch "switch0" \
 		"1:lan" "2:lan" "3:lan" "4:lan" "6@eth1" "5:wan" "0@eth0"
 	;;
+ea8500)
+
+	hw_mac_addr=$(mtd_get_mac_ascii devinfo hw_mac_addr)
+	
+	ucidef_add_switch "switch0" \
+		"1:lan" "2:lan" "3:lan" "4:lan" "0t@eth0" "5:wan" "0t@eth0"
```

Comment:
Does this router really only have one ethernet controller connected to the switch?  Otherwise why not use separate ethernet controllers for wan and lan as for ap148 et al?